_pages/jeffosho.yml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jeffosho
+pageUrl: jeffosho.github.io
+timestamp: 2017-01-09
+---
```

Comment:
@jeffosho you can keep this file, and remove the other ones, but make 2 changes to this one:

1. Change the extension to `.yaml`
1. Add a line break at the end of the file